Source/inv.cpp

```diff
@@ -1430,9 +1430,9 @@ void inv_update_rem_item(int pnum, BYTE iv)
 void CheckInvItem(bool isShiftHeld)
 {
 	if (pcurs >= CURSOR_FIRSTITEM) {
-		CheckInvPaste(myplr, MouseX, MouseY);
+		CheckInvPaste(myplr, { MouseX, MouseY });

```

Comment:
Looking forwared to you converting `MouseX/Y` and `ViewX/Y` with `Point`.

Reply:
Yep.... once the more basic replacements are done, I'll start looking into these shared globals and arrays.